example/src/main/java/io/netty/example/http2/helloworld/client/Http2Client.java

```diff
@@ -77,8 +76,7 @@ public static void main(String[] args) throws Exception {
                     SelectorFailureBehavior.NO_ADVERTISE,
                     // ACCEPT is currently the only mode supported by both OpenSsl and JDK providers.
                     SelectedListenerFailureBehavior.ACCEPT,
-                    SelectedProtocol.HTTP_2.protocolName(),
-                    SelectedProtocol.HTTP_1_1.protocolName()))
+                    "h2", "http/1.1"))
```

Comment:
@trustin I think we should separate them out into their respective codecs:

HTTP/2 constants: https://github.com/netty/netty/blob/master/codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java

SPTY constants: https://github.com/netty/netty/blob/master/codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyCodecUtil.java

WDYT?